dashboard/lib/services/curriculum_pdfs/resources.rb

```diff
@@ -173,7 +173,7 @@ def fetch_resource_pdf(resource, directory = "/tmp/")
         def fetch_url_to_path(url, path)
           service = Drive::DriveService.new
           service.authorization = Google::Auth::ServiceAccountCredentials.make_creds(
-            json_key_io: StringIO.new(CDO.gdrive_export_secret || ""),
+            json_key_io: StringIO.new(JSON.generate(CDO.gdrive_export_secret) || ""),
```

Comment:
```suggestion
            json_key_io: StringIO.new(CDO.gdrive_export_secret.to_json || ""),
```
It seems like these are synonymous and we use to_json a bit more. Is there a reason we should use generate instead?


It also looks like there's a `fast_generate` method. Not for [this](https://ruby-doc.org/stdlib-2.6.3/libdoc/json/rdoc/JSON.html#method-i-fast_generate) PR but maybe worth exploring if this is timing out?

Reply:
Good question.  I noticed that we do use `JSON.generate` in our codebase when I found it, but do see `.to_json` more.  I changed it in this PR.